bucket/mameui.json

```
@@ -0,0 +1,30 @@
+{
+    "homepage": "http://www.mameui.info/",
+    "version": "0.230",
```

The hash changes without notice (it was invalid at the time of testing), so use a nightly version and disable hash verification:

```suggestion
    "version": "nightly",
```